Invalid label storage test

The storage test in the label package doesn't correctly capture the testScenario. Essentially, the label storage test is only ever testing one case.

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Add an error to TestLabels in label/storage_test.go 
2. `make test-go`
3. Notice it only ever runs one test (normally "delete")

__Expected behavior:__
It runs all the tests.

__Actual behavior:__
It only ever runs one of the sub tests.

__Environment info:__

* This happens for all environments.

__Config:__
N/A

__Logs:__

```
GO_TEST_PATHS=./label make test-go
env GOTRACEBACK=all GO111MODULE=on go test -tags 'sqlite_foreign_keys,sqlite_json,' ./label
--- FAIL: TestLabels (0.00s)
    --- FAIL: TestLabels/delete (0.00s)
        logger.go:130: 2022-05-18T11:57:03.769+0100     INFO    Bringing up metadata migrations {"migration_count": 19}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "completed"}
        storage_test.go:224: expected 9 labels, got: 9
    --- FAIL: TestLabels/create (0.00s)
        logger.go:130: 2022-05-18T11:57:03.769+0100     INFO    Bringing up metadata migrations {"migration_count": 19}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "completed"}
        storage_test.go:224: expected 9 labels, got: 9
    --- FAIL: TestLabels/update (0.00s)
        logger.go:130: 2022-05-18T11:57:03.769+0100     INFO    Bringing up metadata migrations {"migration_count": 19}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "completed"}
        storage_test.go:224: expected 9 labels, got: 9
    --- FAIL: TestLabels/list (0.00s)
        logger.go:130: 2022-05-18T11:57:03.769+0100     INFO    Bringing up metadata migrations {"migration_count": 19}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "completed"}
        storage_test.go:224: expected 9 labels, got: 9
    --- FAIL: TestLabels/get (0.00s)
        logger.go:130: 2022-05-18T11:57:03.769+0100     INFO    Bringing up metadata migrations {"migration_count": 19}
        logger.go:130: 2022-05-18T11:57:03.769+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "initial migration", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"userresourcemappingsbyuserindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "migrate task owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create DBRP buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "create pkger stacks buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "delete sessionsv1 bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create TSM metadata buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create Legacy authorization buckets", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "Create legacy auth password bucket", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"telegrafbyorgindexv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "populate dashboards owner id", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "repair DBRP owner and bucket IDs", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.770+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add index \"dbrpbyorgv1\"", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "record shard group durations in bucket metadata", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to operator token", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add annotations and notebooks resource types to all-access tokens", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "repair missing shard group durations", "target_state": "up", "migration_event": "completed"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "started"}
        logger.go:130: 2022-05-18T11:57:03.771+0100     DEBUG   Executing metadata migration    {"migration_name": "add remotes and replications resource types to operator and all-access tokens", "target_state": "up", "migration_event": "completed"}
        storage_test.go:224: expected 9 labels, got: 9
FAIL
FAIL    github.com/influxdata/influxdb/v2/label 0.357s
FAIL
make: *** [GNUmakefile:145: test-go] Error 1
```

__Performance:__
N/A

--- a/label/storage_test.go
+++ b/label/storage_test.go
@@ -221,13 +221,14 @@ func TestLabels(t *testing.T) {
 				}
 
 				if len(l) != 9 {
-					t.Fatalf("expected 2 labels, got: %d", len(l))
+					t.Fatalf("expected 9 labels, got: %d", len(l))
 				}
 			},
 		},
 	}
 
-	for _, testScenario := range tt {
+	for _, ts := range tt {
+		testScenario := ts
 		t.Run(testScenario.name, func(t *testing.T) {
 			t.Parallel()
 

